src/planner_utils.c

```diff
@@ -0,0 +1,71 @@
+#include <postgres.h>
+#include <nodes/plannodes.h>
+#include <miscadmin.h>
+
+#include "planner_utils.h"
+
+static void plantree_walker(Plan **plan, void (*walker) (Plan **, void *), void *ctx);
+
+static inline void
+plantree_walk_subplans(List *plans, void (*walker) (Plan **, void *), void *ctx)
+{
+	ListCell   *lc;
+
+	foreach(lc, plans)
+		plantree_walker((Plan **) &lfirst(lc), walker, ctx);
+}
+
+/* A plan tree walker. Similar to planstate_tree_walker in nodeFuncs.h */
```

Comment:
I guess. But there isn't much of a difference apart from this function walking a Plan tree and the other a PlanState tree (there is pretty much a one-to-one correspondence).